core/services/relay/evm/evm.go

```diff
@@ -811,8 +816,111 @@ type configTransmitterOpts struct {
+	}
+	return accounts, nil
+}
+
+func (l *loopKeystoreAdapter) Sign(ctx context.Context, account string, data []byte) (signed []byte, err error) {
+	// We'll need to convert the tron address to an evm address to sign the data
+	tronAddr, err := tronsdk.Base58ToAddress(account)
+	if err != nil {
+		return nil, err
+	}
+
+	addr := tronAddr.EthAddress()
+	return l.ks.SignRawUnhashedBytes(ctx, addr, data)
+}
+
+type evmTransmissionsCache struct {
+	contractAddress     common.Address
+	client              client.Client
+	contractABI         abi.ABI
+	lp                  logpoller.LogPoller
+	transmittedEventSig common.Hash
+}
+
+func NewEVMTransmissionsCache(ctx context.Context, lggr logger.Logger, contractAddress common.Address, client client.Client, contractABI abi.ABI, lp logpoller.LogPoller, transmittedEventSig common.Hash) *evmTransmissionsCache {
+	return &evmTransmissionsCache{
+		contractAddress:     contractAddress,
+		client:              client,
+		contractABI:         contractABI,
+		lp:                  lp,
+		transmittedEventSig: transmittedEventSig,
+	}
+}
+
+func (c *evmTransmissionsCache) LatestTransmissionDetails(ctx context.Context) (configDigest ocrtypes.ConfigDigest, epoch uint32, round uint8, latestAnswer *big.Int, latestTimestamp time.Time, err error) {
+	// Uses the EVM Client to call the latestConfigDigestAndEpoch function on the contract, reuses the same logic thats in the EVM ContractTransmitter
+	configDigest, epoch, err = getLatestConfigDigestAndEpoch(ctx, c.lp, c.contractAddress, c.contractABI, c.transmittedEventSig, c.client)
+	return configDigest, epoch, 0, latestAnswer, latestTimestamp, nil
+}
+
+func (c *evmTransmissionsCache) LatestRoundRequested(ctx context.Context, lookback time.Duration) (ocrtypes.ConfigDigest, uint32, uint8, error) {
+	return ocrtypes.ConfigDigest{}, 0, 0, nil
+}
+
 // newOnChainContractTransmitter creates a new contract transmitter.
-func newOnChainContractTransmitter(ctx context.Context, lggr logger.Logger, rargs commontypes.RelayArgs, ethKeystore keys.Store, configWatcher *configWatcher, opts configTransmitterOpts, transmissionContractABI abi.ABI, ocrTransmitterOpts ...OCRTransmitterOption) (*contractTransmitter, error) {
+func newOnChainContractTransmitter(ctx context.Context, lggr logger.Logger, rargs commontypes.RelayArgs, ethKeystore keys.Store, configWatcher *configWatcher, opts configTransmitterOpts, transmissionContractABI abi.ABI, ocrTransmitterOpts ...OCRTransmitterOption) (ContractTransmitter, error) {
+	if configWatcher.chain.Config().EVM().ChainType() == chaintype.ChainTron {
+		chainSpecificURL, err := configWatcher.chain.Client().GetExternallyUsedChainSpecificURL(ctx)
+		if err != nil {
+			return nil, err
+		}
+
+		tronClient, err := tronclient.CreateFullNodeClient(chainSpecificURL.URL())
+		if err != nil {
+			return nil, err
+		}
+
+		// Start the Tron TXM
+		tronTXM := trontxm.New(lggr, &loopKeystoreAdapter{ks: ethKeystore}, tronClient, trontxm.TronTxmConfig{})
```

Comment:
Let's be explicit about the configuration here and document that this is currently not configurable for CCIP 1.5 TRON

```
txm.TronTxmConfig{
    BroadcastChanSize: 100,
    ConfirmPollSecs:   2,
}
```